src/Web/Catalog.WebForms/readme.md

```diff
@@ -0,0 +1,80 @@
+# Running the Catalog Editor WebForms application
+
+You can run the catalog editor Web Forms application locally, using
+hard coded catalog data, or in a set of Windows based Docker Containers,
+using the catalog microservice.
+
+To run locally, you edit the [web.config](web.config) file to set the
+`usefake` flag to true:
+
+```xml
+  <appSettings>
+    <add key="usefake" value="true" />
+    <add key ="CatalogURL" value="http://catalog.api:5101" />
+  </appSettings>
+```
+
+Set that value to `false` to run with the full catalog microservice.
+
+Then, set the startup project in Visual Studio to the *Catalog.WebForms* project. Build and run, and test the application.
+
+## Configure Windows Containers
+
+The Catalog.WebForms project uses the full framework, so will only run in Windows based Docker containers. Before running in Docker, make sure you
```

Comment:
Which "full framework" do you mean here? The full Docker framework?

Also, there have been some recent marketing changes that can make Docker versions confusing for people. For that reason, I recommend you try to consistently refer to "Docker for Windows" (which is a Docker Community Edition (Docker CE) version), as consistently as possible when referring to Docker for Windows 10. It may be nice also to link to this doc on Docker for Windows: https://docs.docker.com/docker-for-windows/install/.